### ISSUE TITLE
Insufficient color contrast on the following

### ISSUE THREAD

**Issue by paigethai** [davetyson/project4#37]
<img width="805" alt="image" src="https://user-images.githubusercontent.com/84000119/231210776-bb52a923-f108-4c07-a2a9-6008b49e31bf.png">
- header
- foreground color: #ffffff, background color: #5da9c2 = 2.65

<img width="615" alt="image" src="https://user-images.githubusercontent.com/84000119/231211495-a059c3d2-d8c7-4d89-9f05-54f9cc97fbfd.png">
- search bar / user input
- foreground color: #5da9c1, background color: #f4e6e8 = 2.19

<img width="533" alt="image" src="https://user-images.githubusercontent.com/84000119/231211960-92a43c7e-9fa6-4599-8675-94df284c5ba6.png">
- footer
- foreground color: #5da9c1, background color: #f4e6e8 = 2.19

**Comment by davetyson** [comment]
We definitely need to adjust this! Thanks for leaving such detailed notes!
